JavaScript/Task_Session4/FUnctionCompose.js

```diff
@@ -0,0 +1,14 @@
+//https://www.codewars.com/kata/5655c60db4c2ce0c2e000026/train/javascript
+
+//Function Compose
+
+function compose() {
+  let functionArgs = arguments;
+  return function(param) {
+    let result = param;
+    for (let i = functionArgs.length - 1; i >= 0; i--) {
```

Comment:
using a meaning full name, instead of 'i' you can 'iterator'.
Even the for loop variables should be readable and meaningful